src/fun/counting_v2.go.tmpl

```diff
@@ -0,0 +1,271 @@
+		{{sendMessage nil (cembed 
+			"author" (sdict 
+				"icon_url" (.Guild.IconURL "512") 
+				"name" "🔢 Counting Statistics"
+			) 
+			"description" (printf "⌚ __Current Score__: %d\n🏅 __High Score__: %d on %v by `%s` (%d)\n⏮️ __Last Counter__: `%s` (%d)\n💾 __Saves Remaining__: %d" 
+				(sub $db.next 1) 
+				$db.highscore.num (formatTime $db.highscore.time "01/02") ($user:=userArg $db.highscore.user).Username $user.ID 
+				($user =userArg $db.last.user).Username $user.ID 
+				$db.saves
+			) 
+			"footer" (sdict "text" "Use this command: -CStats [User/\"Leaderboard\"]") 
+			"color" 30654
+		)}}
+		{{return}}
+	{{end}}
+
+	{{if ($user:=index $re2 0 1|toInt|userArg)}} {{/* user stats */}}
+		{{$userCount:=dbGet $user.ID "counting"}}
+		{{if not $userCount}}
+			{{sendMessage nil (cembed 
+				"title" (print "No available stats") 
+				"description" (printf "`%s` has yet to count ☹️\nMaybe give them a heads-up to come join?" 
+					$user.Username
+				) 
+				"footer" (sdict "text" "Use this command: -CStats [User: @/ID]") 
+				"color" 16711680
+			)}}
+		{{else}}
+			{{$userCount:=toInt $userCount.Value}}
+			{{$userCorrect:=(dbGet $user.ID "countingCorrect").Value}}
+			{{sendMessage nil (cembed 
+				"title" (printf "🔢 %s's Counting Statistics" $user.Username) 
+				"description" (printf "%s has counted **%d times**\n**%d** have been correct\nThis makes their average **%.2f%%**" 
+					$user.Mention $userCount (toInt $userCorrect) (div $userCorrect $userCount|mult 100.0)
+				) 
+				"footer" (sdict "text" "Use this command: -CStats [User: @/ID]") 
+				"color" 30654
+			)}}
+		{{end}}
+
+	{{else if eq (index $re2 0 2|lower) "leaderb" "lb"}} {{/* leaderboard */}}
+		{{$desc:=""}}{{$pos:=1}}
+		{{range dbTopEntries "countingCorrect" $leaderboardLength 0}}
+			{{- $desc =printf "%s\n#%-3d %4d - %-4s" $desc $pos (toInt .Value) (or (userArg .UserID) (str .UserID))}}
+			{{- $pos =add $pos 1 -}}
+		{{end}}
+		{{sendMessage nil (cembed 
+			"author" (sdict 
+				"icon_url" (.Guild.IconURL "512") 
+				"name" "Counting Leaderboard"
+			) 
+			"description" (printf "```Pos    ✅  User%s```" $desc) 
+			"footer" (sdict "text" "Use this command: -CStats Leaderboard") 
+			"color" 30654
+		)}}
+
+	{{else}}
+		{{sendMessage nil (cembed 
+			"title" "Invalid Syntax" 
+			"description" "For server counting statistics: `-CStats`\nFor a members' statistics: `-CStats <User: @/ID>`\nFor server leaderboard: `-CStats Leaderboard`" 
+			"color" 16744192
+		)}}
+	{{end}}
+	{{return}}
+{{end}}
+
+{{if and (eq $db.last.user .User.ID) (not $countTwice)}}
+	{{sendMessage nil (cembed 
+		"description" (printf "You can't count twice in a row 🥲\nThe next number is %d" $db.next) 
+		"color" 16744192
+	)}}
+	{{return}}
+{{end}}
+
+{{$invalid:=false}}
+{{$re:=reFindAll `\b([%e]|abs|pi)(\b|$)|[+\-*/^π()]|a?sin|a?cos|a?tan|a?cot|a?sec|a?csc|sqrt|l(o?g|n)|\d+|[MDCLXVI]M{0,3}(CM|CD|D?C{0,3})(XC|XL|L?X{0,3})(IX|IV|V?I{0,3})` .Cmd}}
```

Comment:
```suggestion
{{$re:=reFindAll `\b([%e]|abs|pi)(\b|$)|[+\-*/^π()]|a?(?:sin|cos|tan|cot|sec|csc)|sqrt|l(o?g|n)|\d+|[MDCLXVI]M{0,3}(CM|CD|D?C{0,3})(XC|XL|L?X{0,3})(IX|IV|V?I{0,3})` .Cmd}}
```
Without looking too hard this can be slightly simplified.